src/shared/enums/user-role.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum Role {
```

Comment:
Rather, I'm guided by knowledge from the front, but wouldn't it be better to use a discriminant union instead of `string enum`? All in all, the end result will be the same, it is more about the convention: P

Reply:
It is more of a preference, really. Enums have some advantages like opaque values and easy iteration over keys/values.